vignettes/building-the-dvdrental-docker-image.Rmd

```diff
@@ -0,0 +1,88 @@
+---
+title: "Building the `dvdrental` Docker Image"
+author: "M. Edward (Ed) Borasky, Ian Frantz"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+bibliography: vignettes.bib
+vignette: >
+  %\VignetteIndexEntry{Building the `dvdrental` Docker Image}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+## Introduction
+In the _R, Databases and Docker_ eBook [@Smith2019], we use Docker containers to provide PostgreSQL database services. We use two containers:
```

Comment:
When I generated this vignette, the reference in this sentence ("John David Smith and Muni 2019") did not match the reference in the `Reference` section ("John David Smith, M. Edward (Ed) Borasky, Sophie Yang, and Dipti Muni. 2019."), which does not match the authors ("John David Smith, Sophie Yang, M. Edward (Ed) Borasky, Scott Came, Mary Anne Thygesen, Ian Frantz, and Dipti Muni") at the source (https://smithjd.github.io/sql-pet/). I do not know how the documentation tool is generating these different reference strings, but it would be better if they could all match or follow a common convention, like "Smith et al. 2019".